Notesnook.Inbox.API/src/index.ts

```diff
@@ -184,6 +205,7 @@ app.post("/inbox", async (req, res) => {
 
   const PORT = Number(process.env.PORT || "5181");
   app.listen(PORT, () => {
+    isReady = true;
```

Comment:
This is pointless. The `/health` endpoint won't be available before the server is ready.